CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Consolidated HTTP error variants into a single variant, `HttpError`, which sources directly from the underlying `reqwest::Error` for better error messages.
+- Fixed `RUSTSEC-2020-0071`
```

Comment:
Can you move this up to the "Unreleased" section?

Reply:
Done